low-level/framework/utils/utility.py

```diff
@@ -166,4 +166,3 @@ def errno_to_str_mapping(err_no):
     except Exception as err:
         logger.error("map_errno_to_text errno to text mapping failed due to %s: %s " % err)
         return
-
```

Comment:
nothing changed here, this file should not be there in this PR

Reply:
Yes.. no change in this file. even #L169 is also there at my dev machine.
Still it is showing here